### PR TITLE
improved description

### PR DESCRIPTION
**Pull request by mkaesz-wandb**
## Description

Improves the description of a GORILLA_USE_IDENTIFIER_CLAIMS  env var.

## Ticket

none

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [X] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/hosting/iam/advanced_env_vars.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Choose any of the following environment variables for your instance depending on
...
 | GORILLA_LDAP_GROUP_SYNC | If you are using LDAP based SSO, set it to `true` to automate W&B team membership in your instance based on your LDAP groups. |
 | GORILLA_OIDC_CUSTOM_SCOPES | If you are using OIDC based SSO, you can specify additional [scopes](https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes) that W&B instance should request from your identity provider. W&B does not change the SSO functionality due to these custom scopes in any way. |
-| GORILLA_USE_IDENTIFIER_CLAIMS | If you are using OIDC based SSO, set this variable to `true` to enforce username and full name of your users using specific OIDC claims from your identity provider. If set, ensure that you configure the enforced username and full name in the `preferred_username` and `name` OIDC claims respectively. |
+| GORILLA_USE_IDENTIFIER_CLAIMS | If you are using OIDC based SSO, set this variable to `true` to enforce username and full name of your users using specific OIDC claims from your identity provider. If set, ensure that you configure the enforced username and full name in the `preferred_username` and `name` OIDC claims respectively. Usernames can only contain underscores and hyphens. |
```
Comment (abhinavg6) [inline docs/guides/hosting/iam/advanced_env_vars.md]:
```suggestion
| GORILLA_USE_IDENTIFIER_CLAIMS | If you are using OIDC based SSO, set this variable to `true` to enforce username and full name of your users using specific OIDC claims from your identity provider. If set, ensure that you configure the enforced username and full name in the `preferred_username` and `name` OIDC claims respectively. Usernames can only contain alphanumeric characters along with underscores and hyphens as special characters. |
```

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
Sorry, just requested a minor addition to make it whole.

File: docs/guides/hosting/iam/advanced_env_vars.md (hunk 1 above)
Comment (mkaesz-wandb) [reply docs/guides/hosting/iam/advanced_env_vars.md]: Yeah, makes sense. Thank you.

**Review by abhinavg6 — APPROVED** [review]
LGTM, thanks for doing this

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**